### PR TITLE
ROC-5947: Alert ordnance survey keys

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5947

### Change description

Alert ordnance survery keys not working in app insights

### CODE REVIEW

File: src/test/routes/postcode-lookup.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import 'test/routes/expectations'
+import * as mock from 'nock'
+import { mockPostcodeLookupResponse } from '../data/entity/mockPostcodeLookupResponse'
+
+describe('PostCode Lookup', () => {
+
+  const mockPostcodeServer = 'https://api.ordnancesurvey.co.uk'
+  const mockPostcodePath = /\/places\/v1\/addresses\/postcode\?.+/
+
+  it('should return correct address when postCode lookup ', async () => {
```
Comment (andrewwa-kainos) [inline src/test/routes/postcode-lookup.ts]: Looks like you didn't finish the sentence?

Comment (KiranVarma02) [reply src/test/routes/postcode-lookup.ts]: haha true, let me change that :)

File: src/main/app.ts [hunk 2]
```diff
@@ -125,6 +126,9 @@ app.use((err, req, res, next) => {
     res.render(new ForbiddenError().associatedView)
   } else {
     const view = FeatureToggles.isEnabled('returnErrorToUser') ? 'error_dev' : 'error'
+    if (err.name === 'Template render error') {
+      trackCustomEvent('CMC Dashboard Failure', { error: err })
```
Comment (robertparkinson) [inline src/main/app.ts]: I believe andy is changing this in his PR so might want to check for conflicts: https://github.com/hmcts/cmc-citizen-frontend/pull/2014

Comment (KiranVarma02) [reply src/main/app.ts]: I have @andrewwa-kainos 's commit in my PR, it will not show here once other PR is merged to master.

**Pull request merged**